docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -338,12 +344,12 @@ Here's the list of affected features, colDef flags and props to disable them and
 
 ### CSS classes
 
-- Some CSS classes were removed or renamed
+Some CSS classes were removed or renamed:
 
-  | MUI X v6 classes                            | MUI X v7 classes | Note                   |
-  | :------------------------------------------ | :--------------- | :--------------------- | --- |
-  | `.Mui-hovered`                              | `:hover`         | For rows               |
-  | `.MuiDataGrid--pinnedColumns-(left\|right)` | Removed          | Not applicable anymore | --> |
+| MUI X v6 classes                            | MUI X v7 classes | Note                   |
```

Comment:
I find this table hard to understand

`:hover` is not a class per say

The "Note" column is weirdly named

If we only have 2 classes and with such different use cases, does it even make sense to use a table instead of a list?

Reply:
I don't have any strong preferences as far as using or not the table, but I do think it makes sense to point out the hover change within the CSS classes section—it is a _pseudo_-class in the end, and noting that we have dropped our custom class for the native one is applicable in this place, I think.

Reply:
> but I do think it makes sense to point out the hover change within the CSS classes section

My main concern was to have `MUI X v7 classes` as the column header when in fact no cell was actually containing a proper CSS class :laughing: